src/features/Settings/components/ExcludedAssets.tsx

```diff
@@ -34,6 +36,7 @@ interface ExcludedAssetsProps {
 }
 
 const ExcludedAssets = (props: ExcludedAssetsProps) => {
```

Comment:
Yes. From what I understand, the table was added without going through design and since a mobile view of settings is based on tab navigation, it breaks the layout.